pulpcore.fc

```diff
@@ -1,16 +1,21 @@
-/usr/lib/pulp/bin/gunicorn		--	gen_context(system_u:object_r:pulpcore_exec_t,s0)
+/etc/pulp(/.*)?			gen_context(system_u:object_r:pulpcore_etc_t,s0)
+
+/usr/bin/rq		--	gen_context(system_u:object_r:pulpcore_exec_t,s0)
+/usr/bin/gunicorn	--	gen_context(system_u:object_r:pulpcore_server_exec_t,s0)
```

Comment:
In https://github.com/theforeman/puppet-pulpcore/pull/116 I don't use these and only use wrappers in `/usr/libexec/pulpcore`. Per https://github.com/pulp/pulpcore-selinux/pull/13 I think that's the only place that should actually have a context. Please consider this.

IMHO we should also modify the Ansible based installer to stop using these and only use libexec wrappers. Then we can drop these 2 and `/usr/local/lib/pulp/bin`.

Reply:
In my instance there are no libexec wrappers.

Reply:
@ekohl @zpytela we should keep the `/usr/local/lib/pulp/bin/` filepaths, per the previous convo in a PR (which I finally replied to)
https://github.com/pulp/pulpcore-selinux/pull/13#issuecomment-693544960